frontend/cypress/integration/admin/admin-smoke-tests.spec.js

```diff
@@ -0,0 +1,68 @@
+import {
+  aliasMutation,
+  aliasQuery,
+  setUpCommonGraphqlIntercepts,
+} from "../../support/graphql-test-utils";
+import { getInputByLabel } from "../../support/helpers";
+
+describe("Admin Smoke Tests", () => {
+  const loginViaUI = (role) => {
+    cy.fixture("users.json").then((users) => {
+      const user = users[role];
+      cy.get("input[name=username]").type(user.email);
+    });
+    cy.findByText("Sign-in").click();
+  };
+
+  beforeEach(() => {
+    // common requests
+    setUpCommonGraphqlIntercepts();
+    // page specific requests
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "AllUsers");
+      aliasQuery(req, "User");
+      aliasMutation(req, "UpdateUserAsAdmin");
+    });
+
+    cy.visit("/en/admin");
+  });
+
+  it("Searches for a user, reviews the profile, and edits the phone number", () => {
+    // hit the home page, login, hit the dashboard
+    cy.contains("h1", "Home");
+    cy.contains("a", "Login").click();
+    loginViaUI("admin");
+    cy.contains("h1", "Welcome back");
+    cy.url().should("contain", "/dashboard");
+
+    // find the applicant user to play with
+    cy.contains("a", "Manage users").click();
+    cy.wait("@gqlAllUsersQuery");
+    getInputByLabel("Search").clear().type("Applicant");
+    cy.contains("a", "applicant@test.com").click();
+
+    // exercise profile page
+    cy.contains("h1", "Candidate Details");
+    cy.contains("button", "Print Profile");
+    cy.contains("General Information").click();
+    cy.contains("span", "About");
```

Comment:
This might be a place where exercising aria attributes is the thing to do to know if the tab is selected/loaded?
https://testing-library.com/docs/queries/byrole/#selected

Or maybe it's that it should be finding something in the tab content related to "General Information", which is more indicative of the tab being loaded from the screen-reader perspective?